llvm/lib/Target/AArch64/MCTargetDesc/AArch64MCExpr.h

```diff
@@ -24,6 +24,7 @@ namespace llvm {
 class AArch64MCExpr : public MCTargetExpr {
 public:
   enum VariantKind {
+    // clang-format off
```

Comment:
Ditto

Reply:
Allowing clang-format here breaks custom formatting which IMHO is very nice and helps understanding.

Without clang-format:

```
    VK_CALL              = VK_ABS,
    VK_ABS_PAGE          = VK_ABS      | VK_PAGE,
    VK_ABS_PAGE_NC       = VK_ABS      | VK_PAGE    | VK_NC,
    VK_ABS_G3            = VK_ABS      | VK_G3,
    VK_ABS_G2            = VK_ABS      | VK_G2,
    VK_ABS_G2_S          = VK_SABS     | VK_G2,
    VK_ABS_G2_NC         = VK_ABS      | VK_G2      | VK_NC,
    VK_ABS_G1            = VK_ABS      | VK_G1,
    VK_ABS_G1_S          = VK_SABS     | VK_G1,
    VK_ABS_G1_NC         = VK_ABS      | VK_G1      | VK_NC,
```

With clang-format:

```
    VK_CALL = VK_ABS,
    VK_ABS_PAGE = VK_ABS | VK_PAGE,
    VK_ABS_PAGE_NC = VK_ABS | VK_PAGE | VK_NC,
    VK_ABS_G3 = VK_ABS | VK_G3,
    VK_ABS_G2 = VK_ABS | VK_G2,
    VK_ABS_G2_S = VK_SABS | VK_G2,
    VK_ABS_G2_NC = VK_ABS | VK_G2 | VK_NC,
    VK_ABS_G1 = VK_ABS | VK_G1,
    VK_ABS_G1_S = VK_SABS | VK_G1,
    VK_ABS_G1_NC = VK_ABS | VK_G1 | VK_NC,
```

Can we leave `// clang-format off` here? Or is there probably a better way to teach clang-format how to keep such custom formatting?

Reply:
The main advantage of clang-format is to avoid discussing such things 😄